go.mod

```diff
@@ -4,5 +4,5 @@ go 1.16
 
 require (
 	github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20230719191435-53dea28cff57 // indirect
-	github.com/pulumi/registry/themes/default v0.0.0-20230719165412-e14a803ad211 // indirect
+	github.com/pulumi/registry/themes/default v0.0.0-20230719233533-dc0fd2e0a844 // indirect
```

Comment:
temporarily importing the hugo module from my branch on registry to gen the preview. Will remove this once we are ready to merge.